config/routes.rb

```diff
@@ -1,11 +1,14 @@
 Rails.application.routes.draw do
   devise_for :users
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show new create]
-  end
-  resources :posts, only: %i[index show new create] do
-    resources :comments, only: %i[new create]
-    resources :likes, only: [:create]
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :new, :create, :destroy] do
```

Comment:
- [ ] When I visit the post new path I get this error page. Kindly fix that. See comment below

  ![image](https://github.com/chepkok3/Blog_App/assets/54848363/351adbb1-6440-4247-bac1-e60e8dfd7e77)